search/tests/test_indexing.py

```diff
@@ -245,3 +262,39 @@ def test_term_cache_without_data(self):
         a LearningResource isn't tagged with any terms.
         """
         self.thrice()
+
+    def test_vocab_cache_by_course(self):
+        """
+        Ensure that get_vocabs for one course doesn't
+        pollute the cache for another course.
+        """
+        # Create a resource in a separate course.
+        # pylint: disable=protected-access
```

Comment:
If you mean the pylint hint, it's left over from a previous iteration and is apparently no longer necessary.